### PR TITLE
created Covid19-jumbotron-heb&Eng-versions

### PR DESCRIPTION
**Pull request by Elena-alexeenko**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -353,3 +353,114 @@ hr {
...
+.covid {
+  background-color:#5BC6F9;
+  position: relative;
+}
+
+.covid h1 {
+  color:#fff;
+  text-align: start;
+  padding-top: 2em;
+  padding-left: 6.25em;
```
Comment (soryy708) [inline style.css:360]: Have you considered to instead of using `padding-left` and `padding-right` to then flip them in a `html[dir="rtl"]` to use something like [`padding-inline-start`](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-start) and [`padding-inline-end`](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-end)?

File: style.css [hunk 2]
```diff
@@ -353,3 +353,114 @@ hr {
...
+html[dir='rtl'] .covid .imageCovid {
+  width: 9.25em;
+  height: 9.25em;
+  right:initial;
+  left: 0;
+  bottom: 0;
+  align-items: flex-end;
+}
+
+@media screen and (max-width: 1117px){
```
Comment (soryy708) [inline style.css]: Why add a new one of those instead of using the existing one?

File: style.css [hunk 3]
```diff
@@ -353,3 +353,114 @@ hr {
...
+  .covid {
```
Comment (soryy708) [inline style.css]: Why duplicate code here if it doesn't change anything?

Comment (Elena-alexeenko) [reply style.css]: removed the duplicate

**Comment by soryy708** [comment]
Fix the merge conflict please

File: index.html [hunk 4]
```diff
@@ -121,6 +121,26 @@ <h1>
...
+        <div class = "covid-container">
+        <p>       
+          Qint maximized protection
+          while waiting in line by a contract-free queueing by 
+          system and keeping 
+          distance between 
+          customers while
+          saving their
+          place in line
+        <img src ="assets/Covid.png" class="imageCovid"/>
```
Comment (soryy708) [inline index.html:138]: It's better to use `.svg` where possible

File: hebrew.html [hunk 5]
```diff
@@ -119,6 +119,22 @@ <h1>
...
+      <div class = "covid-content">
+        <h1>
+          COVID-19
+        </h1>
+        <div class = "covid-container">
+          <p>
+            Qint
+            מספקת הגנה מקסימלית עבור לקוחות ועובדים בבית העסק, על ידי ניהול תור ללא מגע תוך כדי שמירה על מרחק בין לקוחות ועל תורם, ובנוסף מאפשרת בקרה על איזון עומסים בכל זמן נתון.
+          </p>
+        <p><img src ="assets/Covid-RTL.png" class="imageCovid"/></p>
```
Comment (soryy708) [inline hebrew.html:132]: Instead of serving an entirely new image, you can `transform: scaleX(-1);` the usual one to flip it.

File: style.css [hunk 6]
```diff
@@ -1,22 +1,43 @@
 html, body {
   margin: 0;
   padding: 0;
-  font-family: 'Poppins', sans-serif;
-  font-size: 16px;
+  font-family: 'Varela Round', sans-serif;
```
Comment (soryy708) [inline style.css:4]: Why revert the changes that were made by someone else?
Did you do a bad merge conflict resolution?

File: style.css [hunk 7]
```diff
@@ -1,22 +1,43 @@
...
+  font-size: 24px;
```
Comment (soryy708) [inline style.css:5]: Why revert the changes that were made by someone else?
Did you do a bad merge conflict resolution?

File: style.css [hunk 8]
```diff
@@ -1,22 +1,43 @@
...
   background-color: #fff;
   width: 100vw;
   overflow-x: hidden;
-  color: #000;
+  color: #424242;
```
Comment (soryy708) [inline style.css:9]: Why revert the changes that were made by someone else?
Did you do a bad merge conflict resolution?

File: style.css [hunk 9]
```diff
@@ -1,22 +1,43 @@
...
+}
+
+@media screen and (max-width: 1380px) {
```
Comment (soryy708) [inline style.css:12]: Why revert the changes that were made by someone else?
Did you do a bad merge conflict resolution?

**Comment by soryy708** [comment]
This branch contains a botched merge conflict resolution which undoes someone else's work.
I can't merge that.

File: style.css (hunk 6 above)
Comment (Elena-alexeenko) [reply style.css:4]: apparently yes
